upstream_utils/upstream_utils.py

```diff
@@ -518,24 +501,21 @@ def format_patch(self):
             ]
         )
 
-        patch_dest = os.path.join(
-            self.wpilib_root, f"upstream_utils/{self.name}_patches"
-        )
-
-        if not os.path.exists(patch_dest):
+        if not os.path.exists(self.get_patch_directory()):
             print(
-                f"WARNING: Patch directory {patch_dest} does not exist", file=sys.stderr
+                f"WARNING: Patch directory {self.get_patch_directory()} does not exist",
```

Comment:
I don't think this warning is necessary. Some upstream libraries have no patches, and Git doesn't commit empty directories, which doesn't seem worthy of a warning.